tigl3/meta.yaml

```diff
@@ -9,7 +9,7 @@ source:
   git_tag: v{{ version }}
 
 build:
-  number: 2
+  number: 0

```

Comment:
What is this number for?

Reply:
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#build-number-and-string You can use it to trigger a rebuild.